specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/examples/CreateApplication.json

```diff
@@ -0,0 +1,61 @@
+{
+    "parameters": {
+        "applicationName": "hue",
+        "clusterName": "cluster1",
+        "resourceGroupName": "rg1",
+        "api-version": "2015-03-01-preview",
+        "subscriptionId": "subid",
+        "parameters": {
+            "computeProfile": {
+                "roles": [{
+                    "name": "edgenode",
+                    "targetInstanceCount": 1,
+                    "hardwareProfile": {
+                        "vmSize": "Standard_D12_v2"
+                    }
+                }]
+            },
+            "installScriptActions": [{
+                "name": "app-install-app1",
+                "uri": "https://.../install.sh",
+                "parameters": "-version latest -port 20000",
+                "roles": [
+                    "edgenode"
+                ]
+            }],
+            "uninstallScriptActions": [],
+            "httpsEndpoints": [{
+                "subDomainSuffix": "dss",
+                "destinationPort": 20000,
+                "accessModes": [
+                    "WebPage"
+                ]
+            }],
+            "provisioningState": "",
+            "applicationType": "CustomApplication",
+            "errors": []
+        }
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "",
+                "name": "",
+                "type": "",
+                "etag": "",
+                "tags": {},
```

Comment:
this examples are useless - please provide real values in the responses

Reply:
Fixed.